decidim-core/app/cells/decidim/user_activity/show.erb

```diff
@@ -6,10 +6,28 @@
       </div>
     <% else %>
       <div class="row">
+        <div class="show-for-mediumlarge filters__left-legend">
+          <%= filter_form_for context[:filter] do |form| %>
+            <div class="columns mediumlarge-6 large-5">
+              <%= form.custom_select :resources,
+                                     :resource_type,
+                                     Decidim::ActionLog.all,
+                                     legend_title: "Show:",
```

Comment:
This "Show" value will be seen in english, whatever the language in which the user is viewing the page.
It could be treated as a translatable tag